core/services/keeper/models.go

```diff
@@ -25,11 +25,11 @@ func (Registry) TableName() string {
 	return "keeper_registries"
 }
 
-// todo - upkeep
 type UpkeepRegistration struct {
```

Comment:
@samsondav I left it in because of the ambiguity over using `upkeepID` - which could be `upkeep.ID` or `upkeep.UpkeepID`. If you just see `upkeepID`, it's not obvious which one it is, but if we keep the model called `UpkeepRegistration`, then we can use `upkeepID` and `upkeepRegID`. What are your thoughts here?